### ISSUE TITLE
--export-plist command in CodeChecker quickcheck 

### ISSUE THREAD

**Issue by gyorb** [Ericsson/codechecker#699]
The changes in #556 are missing because of another pull request where the relevant parts were refactored.

**Comment by whisperity** [comment]
Wasn't these changes discussed to be completely deprecated before even being introduced, because `analyze` already handles this feature?

**Comment by gyorb** [comment]
We might deprecate it but there is a bug fix in that pull request which is still valid and should be in v5.9.
We will discuss the other parts because there will be multiple changes in the next releases.

**Comment by gyorb** [comment]
@whisperity could you handle this issue?

**Comment by whisperity** [comment]
If you specify what was the `bug fix in that pull request which is still valid` (I can't really find it, that pull request was very long ago and I don't remember), then sure.

**Comment by gyorb** [comment]
[This](https://github.com/Ericsson/codechecker/pull/556/commits/c48a04472f776fcc2a1a872ad73b514b33892207) was the bug fix.

**Comment by whisperity** [comment]
[`--keep-tmp` is a **deprecated** argument](https://github.com/Ericsson/codechecker/blob/9cf151c605cf3a3c20ea2f3b3cbfc9ae79f3e9a2/libcodechecker/quickcheck.py#L131-L132) in `quickcheck`.

@gyorb The linked bugfix doesn't make sense anymore, as the `log` and `analyze` part invoked by `quickcheck` does **not** remove any "workspaces" anymore.

**Comment by whisperity** [comment]
Okay so basically. There was an issue, that even if you supplied `--keep-tmp`, it would **still** remove the temporary folder. The bugfix fixed this, so if `--keep-tmp` was given, it would _actually_ **not** delete the folder. The fix was implemented because `--keep-tmp`'s help mentioned its expected behaviour.

Earlier in the restructuring, we decided that (seeing that it *never* actually worked) the argument would be deprecated from the valid argument list. Thus, the fix is also obsolete.

Current behaviour is that the temp folder (`/tmp/codechecker-qc-[some random stuff]`) is automatically deleted, and specifying `--keep-tmp` is indifferent to the invocation.
